app/assets/styles/app.scss

```diff
@@ -122,3 +126,29 @@ p, label, input {
 	color: $red;
 }
 
+//Footer
+footer {
+	bottom: 0;
+	width: 100%;
+	text-align: right;
+	padding: 1em;
+	margin-top: 5%;
+	background-color: $dark-teal;
+}
+
+#h4i_logo {
+	content: url("../../static/images/h4i_square_white.png");
```

Comment:
If the logos are used more than once, would consider setting the png urls as a sass variables